libraries/stdlib/samples/test/samples/text/strings.kt

```diff
@@ -454,4 +454,12 @@ class Strings {
         assertPrints(inputString0.replace('s', 'z'), "Mizzizzippi")
         assertPrints(inputString1.replace("data", "information"), "Insufficient information for meaningful answer.")
     }
+
+    @Sample
+    fun startsWith() {
+        val string = "Silly Goose Egg";
+        assertTrue(string.startsWith('S'));
+        assertTrue(string.startsWith('s'));
+        assertFails(string.startsWith('a');
```

Comment:
This should be `assertFalse`, not `assertFails`. Also, it appears you are missing a closing `)`